stubs/biopandas/METADATA.toml

```diff
@@ -0,0 +1,11 @@
+version = "0.4.*"
+upstream_repository = "https://github.com/rasbt/biopandas"
+# requires a version of numpy with a `py.typed` file
+# TODO: Lots of stubtest errors when using numpy 2
+requires = ["numpy>=1.20,<2", "pandas-stubs"]
+partial_stub = true
+
+[tool.stubtest]
+ignore_missing_stub = true
+# stub_uploader won't allow pandas-stubs in the requires field https://github.com/typeshed-internal/stub_uploader/issues/90
```

Comment:
this comment seems out of place, since it's in the stubtest section. stubtest is an internal tool for typeshed's tests and doesn't have stub_uploader's security concerns.